src/test/java/com/exasol/adapter/jdbc/ColumnMetadataReaderTest.java

```diff
@@ -86,8 +85,8 @@ void testParseBoolean() throws SQLException, RemoteMetadataReaderException {
 
     private void assertSqlTypeConvertedToExasolType(final int typeId, final DataType expectedDataType)
             throws SQLException {
-        mockDatatype(typeId);
-        assertThat(mapSingleMockedColumn().getType(), equalTo(expectedDataType));
+        when(this.columnsMock.getInt(DATA_TYPE_COLUMN)).thenReturn(typeId);
+        assertThat(mapSingleMockedColumn("DOUBLE").getType(), equalTo(expectedDataType));
```

Comment:
The "DOUBLE" constant looks suspicious here. That can't be right.

Reply:
It's "original type name" I think